apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx

```diff
@@ -57,6 +59,9 @@ export const useMultipleBannerConfig = () => {
     const SHUFFLE_BANNERS: IBannerConfig[] = [
       { shouldRender: true, banner: <LineaBanner /> },
       { shouldRender: true, banner: <ArbitrumOneBanner /> },
+      { shouldRender: true, banner: <GalxePedictBanner /> },
+      { shouldRender: true, banner: <GalxeSyndicateBanner /> },
```

Comment:
Move both to `NO_SHUFFLE_BANNERS ` array, and make them before `<UserBanner/> `.